platforms/Windows/python/python.wxi

```diff
@@ -0,0 +1,40 @@
+﻿<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
+  <?define PythonRoot = "$(ImageRoot)\Python-$(PythonVersion)"?>
+
+  <Fragment>
+    <DirectoryRef Id="INSTALLROOT">
+      <Directory Id="Python" Name="Python-$(PythonVersion)" >
+        <Directory Id="python_licenses" Name="licenses" />
+      </Directory>
+    </DirectoryRef>
+  </Fragment>
+
+  <Package
+    Language="1033"
+    Manufacturer="!(loc.ManufacturerName)"
+    Name="!(loc.EmbeddedPython_ProductName)"
+    UpgradeCode="$(PythonUpgradeCode)"
+    Version="$(PythonVersion)"
+    Scope="$(PackageScope)">
+
+    <Media Id="1" Cabinet="python.cab" EmbedCab="$(ArePackageCabsEmbedded)" />
+
+    <WixVariable Id="SideBySidePackageUpgradeCode" Value="$(PythonUpgradeCode)" />
+    <FeatureGroupRef Id="SideBySideUpgradeStrategy" />
+
+    <ComponentGroup Id="EmbeddedPython" Directory="Python">
+      <Files Include="$(PythonRoot)/**" />
```

Comment:
Please explicitly list the files. This is to ensure that we know _exactly_ what is being shipped for licensing purposes.